stable/prometheus-cloudflare-exporter/templates/service.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "prometheus-cloudflare-exporter.fullname" . }}
+  labels:
+{{ include "prometheus-cloudflare-exporter.labels" . | indent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: 80
```

I think it's conventional for exporters to use the same port

```suggestion
    - port: 9199
```